coa-helm/templates/identity.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: "aadpodidentity.k8s.io/v1"
```

Comment:
This creates the `AzureIdentity` and `AzureIdentityBinding` objects, with values supplied by Leo.